components/molecules/charts/highCharts/bigData/__tests__/stories.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * highCharts - big data line chart
+ */
+
+// UI
+import { HighChart } from 'components'
+import Readme from '../README.md'
+
+// Data
+import { BigDataDataSource } from './initialDataSource'
+
+// Styled Components
+import styled from 'styled-components'
+
+export default {
+  args: {
+    enableExport: false,
+    titleText: 'Big data',
+    tooltipTitle: 'data',
+    violationMin: 24.5,
+    violationMax: 26.5,
+    withViolationLimits: true
+  },
+  argTypes: {
+    violationMin: { control: { type: 'range', min: 0, max: 100, step: 0.5 } },
+    violationMax: { control: { type: 'range', min: 0, max: 100, step: 0.5 } }
+  },
+  title: 'Molecules/Charts/highCharts/BigData',
+  component: HighChart,
+  parameters: {
+    docs: {
+      description: {
+        component: Readme
+      }
+    }
+  }
+}
+
+const BaseComponent = props => {
+  const violationCheck = ({ y }) => {
+    if (props.args.withViolationLimits) {
+      if (y > props.args.violationMax || y < props.args.violationMin) {
+        return '#ee00ee'
+      }
+    }
+  }
+
+  const options = {
+    chart: {},
+
+    title: {
+      text: props.args.titleText
+    },
+
+    exporting: {
+      enabled: props.args.enableExport
+    },
+    series: [
+      {
+        name: props.args.tooltipTitle,
+        data: BigDataDataSource.map((item, index) => {
+          return {
+            x: Date.parse(item.x),
+            y: Number(item.y),
+            marker: { fillColor: violationCheck({ y: item.y }) }
+          }
+        }),
+        marker: {
+          enabled: true,
+          lineWidth: 1,
+          lineColor: null
+        }
+      }
+    ]
+  }
+
+  return (
+    <StyledWrapper>
+      <HighChart options={options} constructorType='stockChart' />
+    </StyledWrapper>
+  )
+}
+
+const StyledWrapper = styled.div`
+  text-align: center;
+  height: 500px;
+`
+/// storeis
```

Comment:
@mohsenParvar I have seen a few of these in different files - can you remove please

Somes notes
Space out code some more between methods and logical breaks - its easier to read I believe

DocsBlocks - please add these in with the comments on imports - its then easy to quickly see what our dependancy blocks are - I like this for again quick viewing

Have one more pass on this then I think good to go - thanks